### PR TITLE
Move depends and description to single line

### PR DESCRIPTION
**Pull request by jlongever**
@RackHD/corecommitters @RackHD/rackhd_dev

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -7,13 +7,6 @@ Standards-Version: 3.9.4
...
-Depends: ${shlibs:Depends}, ${misc:Depends},
- nodejs,
- nodejs-legacy,
- npm,
- mongodb,
- snmp,
- ipmitool
+Depends: ${shlibs:Depends}, ${misc:Depends}, nodejs, nodejs-legacy, npm, mongodb, snmp, ipmitool
 Suggests: python-pywbem, ansible, apt-mirror, amtterm
-Description: on-taskgraph
```
Comment (heckj) [inline debian/control]: does this build out correctly with making the debian package? Just not sure re: change to Description data

Comment (jlongever) [inline debian/control]: I don't think Package name has to be in the Description?  Looks like it built up correctly:

```
~/on-taskgraph/packagebuild$ dpkg-deb --info ../on-taskgraph_2016-01-04-194655Z_amd64.deb
 new debian package, version 2.0.
 size 5599380 bytes: control archive=166491 bytes.
      28 bytes,     1 lines      conffiles
     450 bytes,    10 lines      control
  932481 bytes,  6101 lines      md5sums
     451 bytes,    13 lines   *  postinst             #!/bin/sh
     343 bytes,    11 lines   *  preinst              #!/bin/sh
     219 bytes,     7 lines   *  prerm                #!/bin/sh
 Package: on-taskgraph
 Version: 2016-01-04-194655Z
 Architecture: amd64
 Maintainer: Joe Heck <joseph.heck@emc.com>
 Installed-Size: 36936
 Depends: libc6 (>= 2.14), libgcc1 (>= 1:4.1.1), libstdc++6 (>= 4.1.1), sysv-rc (>= 2.88dsf-24) | file-rc (>= 0.8.16), nodejs, nodejs-legacy, npm, mongodb, snmp, ipmitool
 Suggests: python-pywbem, ansible, apt-mirror, amtterm
 Section: utilties
 Priority: optional
```

Comment (heckj) [inline debian/control]: Cool, thanks!

**Comment by heckj** [comment]
:+1:

**Comment by anhou** [comment]
:+1:

**Pull request merged**